src/search/lp/lp_internals.cc

```diff
@@ -118,12 +118,10 @@ unique_ptr<OsiSolverInterface> create_lp_solver(LPSolverType solver_type) {
         break;
     case LPSolverType::CPLEX:
 #ifdef COIN_HAS_CPX
-        {
-            OsiCpxSolverInterface *cpx_solver = new OsiCpxSolverInterface;
-            CPXsetintparam(cpx_solver->getEnvironmentPtr(), CPX_PARAM_THREADS, 1);
-            cpx_solver->passInMessageHandler(new ErrorCatchingCoinMessageHandler);
-            lp_solver = cpx_solver;
-        }
+        OsiCpxSolverInterface *cpx_solver = new OsiCpxSolverInterface;
```

Comment:
I'm sorry, but I think I remember now why we need the braces here and it has nothing to do with style hacks or uncrustify: If only one of the solvers is installed, there was an issue with variable scopes. I don't see a problem currently, so maybe this was fixed by the `#ifdef`s. The tests compile with and without CPLEX, right? So if anything goes wrong, we'll catch it there.